### DISCUSSION TITLE
different plot in plotPDF

### DISCUSSION THREAD

**Discussion by amirtopaz** [mtex-toolbox/mtex#1215] [Ask Anything]
Hi everybody,
the following command - plot(grains,grains.longAxis.rho/degree) leads to a map where the grains are colored according to their shape orientation. 
Is there a way to do the same thing but with the plotPDF command? 
I want to plot the CPO of grains that will be colored according to their long axis orientation.

I did something similar by manually inserting grains into arrays of constant range according to the long axis direction, and then plotted them separately in the plotPDF. 
Instead of colored dots inserted manually, I want it to look more like a heat map or contoured according to the shape preferred orientation. Is it possible?

Thanks,
Amir

![image](https://user-images.githubusercontent.com/62837798/134345618-11cec0cb-ebc4-4883-aac5-2db30a8bc3aa.png)

**Comment by kilir** [comment]
Hi,
for `plotPDF` as in (almost) any other plot type, data needs to be supplied as the second input argument. See here: https://mtex-toolbox.github.io/orientation.plotPDF.html
Rüdiger
